build.sbt

```diff
@@ -1,38 +1,39 @@
-crossScalaVersions := Seq("2.10.5", "2.11.8")
+val baseSettings = Seq(
+  organization := "com.lihaoyi",
+  name := "fansi",
+  version := "0.2.3",
+
+  scalaVersion := "2.11.8",
+  crossScalaVersions := Seq("2.10.6", "2.11.8", "2.12.0-RC2"),
+  
+  scmInfo := Some(ScmInfo(
+    browseUrl = url("https://github.com/lihaoyi/utest"),
+    connection = "scm:git:git@github.com:lihaoyi/utest.git"
+  )),
+  licenses := Seq("MIT" -> url("http://www.opensource.org/licenses/mit-license.html")),
+  developers += Developer(
+    email = "haoyi.sg@gmail.com",
+    id = "lihaoyi",
+    name = "Li Haoyi",
+    url = url("https://github.com/lihaoyi")
+  )
+)
+
+baseSettings
 
 lazy val fansi = crossProject
+  .settings(baseSettings)
   .settings(
-    scalaVersion := "2.11.8",
-    organization := "com.lihaoyi",
-    name := "fansi",
-    version := "0.2.2",
-    scalacOptions += "-target:jvm-1.7",
+    scalacOptions ++= Seq(scalaBinaryVersion.value match {
+      case x if x.startsWith("2.12") => "-target:jvm-1.8"
+      case _ => "-target:jvm-1.7"
+    }),
     libraryDependencies ++= Seq(
-      "com.lihaoyi" %%% "utest" % "0.4.3" % "test",
-      "com.lihaoyi" %%% "sourcecode" % "0.1.1"
+      "com.lihaoyi" %%% "sourcecode" % "0.2.0",
```

Comment:
@lihaoyi 
it should be 0.1.2 based on https://github.com/lihaoyi/sourcecode#sourcecode--